tests/lib/rules/no-restricted-properties.js

```diff
@@ -75,6 +75,21 @@ ruleTester.run("no-restricted-properties", rule, {
                 object: "obj",
                 property: "foo"
             }]
+        }, {
+            code: "foo.bar",
+            options: [{
+                property: "baz"
+            }]
+        }, {
+            code: "foo.bar",
+            options: [{
+                object: "baz"
+            }]
+        }, {
+            code: "foo()",
```

Comment:
No error, because no property of `foo` is being accessed. Should I add a test for that?